README.md

```diff
@@ -264,6 +265,7 @@ Table of Contents
   * [microenv.com](https://microenv.com) —  Create fake REST API for developers with possibility to generate code and app in docker container.
   * [neptune.ai](https://neptune.ai/) - Log, store, display, organize, compare and query all your MLOps metadata. Free for individuals: 1 member, 100 GB of metadata storage, 200h of monitoring/month
   * [News API](https://newsapi.org) — Search news on the web with code, get JSON results. Developers get 3,000 queries free each month.
+  * [nsfw.rest](https://nsfw.rest) - Free API to scan images for NSFW content and files for malware. 100 request per second.
```

Comment:
there doesnt seem much point to this, the unscan one is by the same people and does more, is this one just there for SEO or something?